src/main/java/stringCalculator/Operation.java

```diff
@@ -0,0 +1,41 @@
+package stringCalculator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BinaryOperator;
+
+public interface Operation {
+    String apply(String a, String b);
+}
+
+enum FourOperation implements Operation {
+    PLUS("+", (a,b) -> (Integer.parseInt(a)+Integer.parseInt(b))+""),
+    MINUS("-", (a,b) -> (Integer.parseInt(a)-Integer.parseInt(b))+""),
+    TIMES("*", (a,b) -> (Integer.parseInt(a)*Integer.parseInt(b))+""),
+    DIVIDE("/",(a,b) -> (Integer.parseInt(a)/Integer.parseInt(b))+"");
```

Comment:
```suggestion
    PLUS("+", (a,b) -> a + b),
    MINUS("-", (a,b) -> a - b),
    TIMES("*", (a,b) -> a * b),
    DIVIDE("/",(a,b) -> a / b);
```
변환은 다른 객체에게 위임해주면 어떨까요?